src/reducers/slidebar.js

```diff
@@ -0,0 +1,20 @@
+const INITIAL_STATE = {
+  slide_order: null
+};
+
+const slidebarReducer = (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+    case 'CHANGE_SLIDE_ORDER':
+        let newState = state.toJS();
```

Comment:
Not having looked into it, I'm not sure if there's a better way than doing this -- this might be expensive with n number of slides needing to be run through .toJS() before anything else happens here

Reply:
This reducer is not used and has been deleted.